components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -428,21 +428,41 @@ bool PublisherInfoDatabase::CreateActivityInfoTable() {
   std::string sql;
   sql.append("CREATE TABLE ");
   sql.append(name);
-  sql.append(
-      "("
-      "publisher_id LONGVARCHAR NOT NULL,"
-      "duration INTEGER DEFAULT 0 NOT NULL,"
-      "visits INTEGER DEFAULT 0 NOT NULL,"
-      "score DOUBLE DEFAULT 0 NOT NULL,"
-      "percent INTEGER DEFAULT 0 NOT NULL,"
-      "weight DOUBLE DEFAULT 0 NOT NULL,"
-      "reconcile_stamp INTEGER DEFAULT 0 NOT NULL,"
-      "CONSTRAINT activity_unique "
-      "UNIQUE (publisher_id, reconcile_stamp) "
-      "CONSTRAINT fk_activity_info_publisher_id"
-      "    FOREIGN KEY (publisher_id)"
-      "    REFERENCES publisher_info (publisher_id)"
-      "    ON DELETE CASCADE)");
+  if (currentVersion) {
```

Comment:
could you maybe just append this two lines into the whole string if needed, this way we are less error prone

Reply:
Done.